libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -24,41 +24,38 @@
 from .links import Link
 from .math import cosine_similarity
 
-CONTENT_ID = "content_id"
-
 
 @dataclass
 class Node:
     """Node in the GraphStore"""
 
     id: Optional[str] = None
     """Unique ID for the node. Will be generated by the GraphStore if not set."""
+    text: str = None
```

Comment:
I wonder if we should call this `content`. In the future, we could change it to `Union[str, bytes]` to support different mime types with inline images or not, etc. That said, I think that can be evolved later if needed.

Reply:
Yes we can see that later.